node/test_helpers.go

```diff
@@ -117,16 +117,21 @@ func waitForAtLeastNBlocks(node Node, n int, source Source) error {
 	})
 }
 
-// TODO: use n and return n validators
```

Comment:
I think this is a right direction long term for this helper, but in the short term we only support 1 validator right? Or is it multiple validators, but 1 sequencer?

Reply:
Agreed, should just support 1 validator. The validator set is the same as the sequencer set here.

Reply:
Agreed, I think it's best to remove `n` argument and just return a validator set containing one validator always.